src/pgduckdb_hooks.cpp

```diff
@@ -52,7 +52,11 @@ ContainsDuckdbFunctions(Node *node, void *context) {
 
 	if (IsA(node, Query)) {
 		Query *query = (Query *)node;
-		return query_tree_walker(query, ContainsDuckdbFunctions, context, 0);
+#if PG_VERSION_NUM >= 160000
+	return query_tree_walker(query, ContainsDuckdbFunctions, context, 0);
```

Comment:
nitpick: This needs another indent (same for block below), to make it clear that it's part of the `if`